python-for-loop/break_continue.py

```diff
@@ -0,0 +1,14 @@
+numbers = [1, 3, 5, 7, 9]
+target = 5
+for number in numbers:
+    print(f"Processing {number}...")
+    if number == target:
+        print(f"Target found {target}!")
+        break
+
+numbers = [1, 2, 3, 4, 5, 6]
+for number in numbers:
+    print(f"{number = }")
```

Comment:
flake8 compains and I was also wondering about it, I've generally only seen this used without whitespace:

```suggestion
    print(f"{number=}")
```

`pycodestyle`, which implements the check, seems to be stubborn about keeping it this way: https://github.com/PyCQA/pycodestyle/issues/1201

```sh
$ flake8 break_continue.py 
break_continue.py:11:20: E251 unexpected spaces around keyword / parameter equals
break_continue.py:11:22: E202 whitespace before '}'
break_continue.py:11:22: E251 unexpected spaces around keyword / parameter equals
```

But I know you're using space around the equals sign also [in the f-string tutorial](https://realpython.com/python-f-strings/#self-documenting-expressions-for-debugging).

So IDK, I guess the linters don't fail on it?

If you decide to change it, you'd also have to change it in the written tutorial.

Reply:
If I remember correctly, in one of my previous articles I used this feature without spaces and I guess it was Geir Arne who suggested adding the spaces for readability. However, I don't think this is a written rule.

Reply:
Ok, yeah I'm fine with leaving it considering it passes our linter checks. @gahjelle should we note this somewhere, that we'll ignore E251 and E202 for these cases?